test/test_reductions.py

```diff
@@ -3150,12 +3150,10 @@ def test_amax_amin_zero_dim_tensors(self, device):
 
             # Tests to check if reduction happens along the specified dim.
             self.assertEqual(torch.empty((2, 0), device=device), fn(master_input, dim=2))
```

Comment:
This looks redundant since we are already checking against numpy.

Reply:
Agreed! This makes sense to me too... Thanks!

Actually, I had two motives while writing tests for the function:
1. To ensure it returns an empty tensor.
2. To ensure its behavior is similar to NumPy.
So, I wrote it...